### PR TITLE
feat: show flow type in the table

### PR DESCRIPTION
**Pull request by abuaboud**
## What does this PR do?

Fixes # (issue)

### CODE REVIEW

File: packages/backend/.env
Comment (AbdulTheActivePiecer) [inline packages/backend/.env:1]: why is AP_TEMPLATES_SOURCE_URL being deleted ?

File: packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.html [hunk 1]
```diff
@@ -35,14 +35,18 @@
...
               <td mat-cell *matCellDef="let flow">
-                <div class="ap-flex ap-gap-2">
+                <div class="ap-flex ap-gap-2 ap-items-center ap-justify-center">
                   <mat-slide-toggle color="primary" [class.ap-cursor-not-allowed]="flow.instanceToggleControl.disabled"
                     (click)="$event.stopImmediatePropagation(); toggleFlowStatus(flow,flow.instanceToggleControl)"
                     [formControl]="flow.instanceToggleControl" [matTooltip]="flow.instanceToggleControl.disabled? 'Please publish the flow':
                   flow.instanceToggleControl.value? 'Flow is on' : 'Flow is off'"></mat-slide-toggle>
-                  <div [class.ap-invisible]=" !flowsUpdateStatusRequest$[flow.id]" class="spinner-border dark-spinner"
-                    role="status">
-                  </div>
```
Comment (AbdulTheActivePiecer) [inline packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.html]: Why did you remove this loader? it is meant to show up when someone switches off/on the instance

File: packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.ts [hunk 2]
```diff
@@ -162,6 +167,32 @@ export class FlowsTableComponent implements OnInit {
...
+  getTriggerIcon(flow: Flow) {
+    const trigger = flow.version.trigger;
+    switch (trigger.type) {
+      case TriggerType.WEBHOOK:
+        return 'assets/img/custom/triggers/instant.svg';
+      case TriggerType.PIECE: {
+        return 'assets/img/custom/triggers/periodic.svg';
+      }
+      case TriggerType.EMPTY:
+        throw new Error("Flow can't be published with empty trigger");
```
Comment (AbdulTheActivePiecer) [inline packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.ts:185]: no need to throw an error, just don't show anything by returning '';

File: packages/backend/.env [hunk 3]
```diff
@@ -24,4 +24,3 @@ AP_WARNING_TEXT_HEADER=""
 AP_FLOW_WORKER_CONCURRENCY="5"
 
 AP_DEV_PIECES="discord,delay,hackernews,github,slack,store,wordpress"
```
Comment (AbdulTheActivePiecer) [inline packages/backend/.env:26]: why remove this ?

Comment (abuaboud) [inline packages/backend/.env:26]: It doesn't work in the development mode.

File: packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.ts (hunk 2 above)
Comment (abuaboud) [reply packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.ts:185]: It case shouldn't happen, both are okay.

File: packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.html (hunk 1 above)
Comment (abuaboud) [reply packages/ui/feature-dashboard/src/lib/pages/flows-table/flows-table.component.html]: Can you handle this?
